app/routers/symbol_settings.py

```
@@ -0,0 +1,823 @@
+    class Config:
+        from_attributes = True
+
+
+# ==========================================
+# 사용자 기본 설정 API 엔드포인트
+# ==========================================
+
+@router.get("/user-defaults", response_model=UserTradeDefaultsResponse)
+async def get_user_defaults(
+    request: Request,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)